docs/linking/4-grant-consent.puml

```diff
@@ -14,11 +14,14 @@ participant "DFSP" as DFSP
 
 autonumber 1 "<b>GRANT-#</b>"
 
-DFSP -> Switch ++: ""POST /consents {""\n\
+DFSP -> Switch ++: ""POST /consents""\n\
+""  FSIOP-Source: dfspa""\n\
+""  FSIOP-Destination: auth.dfspa**""\n\
```

Comment:
do we have some extra syntax format to express it or is this example only for illustrative purpouses